python/lib/database_lib/visit_windows.py

```diff
@@ -44,6 +44,5 @@ def check_visit_label_exits(self, visit_label):
         """
 
         query = 'SELECT Visit_label FROM Visit_Windows WHERE BINARY Visit_label = %s'
-        results = self.db.pselect(query=query, args=(visit_label,))
-
-        return results if results else None
+        results = self.db.pselect(query=query, args=[visit_label])
```

Comment:
I think that would break since pselect args expects a tuple.

Reply:
The code worked fine (on my machine™) with the lists and I saw a few places that use lists (in `lib/candidate.py`) so I went with it. But it seems that even the MySQL docs uses tuples so you're probably right that I should go with it yes. (although I must say I do not get why one would design this API with tuples over lists here)